app/src/pages/Register.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+const Register = () => {
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [repeatPassword, setRepeatPassword] = useState("");
+
+  const handleSubmit = (formEvent: React.formEvent) => {
+    formEvent.preventDefault();
+  };
```

Comment:
@majamitic774 This function doesn't return anything. So add return type `void` for the function.
```typescript
const handleSubmit = (
  formEvent: React.FormEvent<HTMLFormElement>): void => { 
  formEvent.preventDefault();
};
```